keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -693,9 +693,12 @@ in back-to-back releases.
 
```

Comment:
Minor nit: the appropriate boxes in `## Release Signoff Checklist` should be checked :wink: at the beginning of the doc.

Reply:
In the `## Summary` section, can we use official runc's doc link rather than a PR to a feature?

Reply:
Also, the summary should serve as a one sentence description of what this enhancement is about. Currently the goals section does it better than the summary. I'd probably move all the text from summary under motivation, and make the summary short and concise. See https://www.youtube.com/watch?v=JZ9LQR_j0Rk&t=1482s for reference.

Reply:
Lastly, it seems the feature is targetted at more CRIs, not runc only, so can we make that clear from both the summary and motivation section. I see below that beta is being expanded with additional runtimes, so I'd like to see the feature be global, rather than only available in a single CRI implementation.

Reply:
> Minor nit: the appropriate boxes in `## Release Signoff Checklist` should be checked 😉 at the beginning of the doc.

Done

Reply:
> In the `## Summary` section, can we use official runc's doc link rather than a PR to a feature?

Done, but s/Summary/Motivation/, s/runc/OCI Runtime Spec/

Reply:
> Also, the summary should serve as a one sentence description of what this enhancement is about. Currently the goals section does it better than the summary. I'd probably move all the text from summary under motivation, and make the summary short and concise. See https://www.youtube.com/watch?v=JZ9LQR_j0Rk&t=1482s for reference.

Done.

**Summary**: Make read-only volumes recursively read-only. e.g., if `/mnt` is mounted as read-only, its submounts such as `/mnt/usbstorage` should be read-only too.

Reply:
> Lastly, it seems the feature is targetted at more CRIs, not runc only, so can we make that clear from both the summary and motivation section. I see below that beta is being expanded with additional runtimes, so I'd like to see the feature be global, rather than only available in a single CRI implementation.

CRI runtimes implemented the RRO mount only after this KEP gets accepted, so it is illogical to mention them in the "Summary"/"Motivation" section